set.mm

```diff
@@ -76402,19 +76402,18 @@ any sets (which usually are functions) and any element (even not
        12-Apr-2019.) $)
     suppsnop $p |- ( ( X e. V /\ Y e. W /\ Z e. U )
                    -> ( F supp Z ) = if ( Y = Z , (/) , { X } ) ) $=
-      ( vx wcel csn wne c0 cif wceq cvv 3adant3 a1i syl2anc neeq1d w3a csupp co
-      cv cima cdm crab wf cop wa wf1o f1osng f1of syl feq1i sylibr snex suppval
-      fex simp3 dmeqd dmsnopg 3ad2ant2 eqtrd sneq imaeq2d rabsnif syl6eq wn cfv
-      rabeq fnsng eqcomi fneq1d mpbid snidg 3ad2ant1 fnsnfv eqcomd fveq1i fvsng
-      wfn syl5eq sneqd wb sneqbg necon3abid 3bitrd ifbid ifnot 3eqtrd ) ECJZFDJ
-      ZGAJZUAZBGUBUCZBIUDZKZUEZGKZLZIBUFZUGZBEKZUEZWTLZXDMNZFGOZMXDNZWOBPJZWNWP
-      XCOWOXDFKZBUHZXDPJZXJWOXDXKEFUIKZUHZXLWLWMXOWNWLWMUJXDXKXNUKXOEFCDULXDXKX
-      NUMUNQXDXKBXNHUOUPXMWOEUQRXDXKPBUSSWLWMWNUTIPABGURSWOXCXAIXDUGZXGWOXBXDOX
-      CXPOWOXBXNUFZXDWOBXNBXNOWOHRVAWMWLXQXDOWNEFDVBVCVDXAIXBXDVKUNXAXFIEWQEOZW
-      SXEWTXRWRXDBWQEVEVFTVGVHWOXGXHVIZXDMNXIWOXFXSXDMWOXFEBVJZKZWTLXKWTLXSWOXE
-      YAWTWOBXDWBZEXDJZXEYAOWOXNXDWBZYBWLWMYDWNEFCDVLQWOXDXNBXNBOWOBXNHVMRVNVOW
-      LWMYCWNECVPVQYBYCUJYAXEXDEBVRVSSTWOYAXKWTWOXTFWOXTEXNVJZFEBXNHVTWLWMYEFOW
-      NEFCDWAQWCWDTWOXHXKWTWMWLXKWTOXHWEWNFGDWFVCWGWHWIXHXDMWJVHWK $.
+      ( vx wcel csn cima wne crab c0 cif wceq cvv wf neeq1d w3a csupp co cv cdm
+      cop wa wf1o f1osng f1of syl 3adant3 feq1i sylibr snex fex sylancl suppval
+      simp3 syl2anc fdmd rabeqdv sneq imaeq2d rabsnif syl6eq cfv wfn ffnd snidg
+      3ad2ant1 fnsnfv eqcomd fveq1i fvsng syl5eq sneqd sneqbg necon3abid 3bitrd
+      wn wb 3ad2ant2 ifbid ifnot 3eqtrd ) ECJZFDJZGAJZUAZBGUBUCZBIUDZKZLZGKZMZI
+      BUEZNZBEKZLZWOMZWSOPZFGQZOWSPZWJBRJZWIWKWRQWJWSFKZBSZWSRJXEWJWSXFEFUFKZSZ
+      XGWGWHXIWIWGWHUGWSXFXHUHXIEFCDUIWSXFXHUJUKULWSXFBXHHUMUNZEUOWSXFRBUPUQWGW
+      HWIUSIRABGURUTWJWRWPIWSNXBWJWPIWQWSWJWSXFBXJVAVBWPXAIEWLEQZWNWTWOXKWMWSBW
+      LEVCVDTVEVFWJXBXCWAZWSOPXDWJXAXLWSOWJXAEBVGZKZWOMXFWOMXLWJWTXNWOWJBWSVHZE
+      WSJZWTXNQWJWSXFBXJVIWGWHXPWIECVJVKXOXPUGXNWTWSEBVLVMUTTWJXNXFWOWJXMFWJXME
+      XHVGZFEBXHHVNWGWHXQFQWIEFCDVOULVPVQTWJXCXFWOWHWGXFWOQXCWBWIFGDVRWCVSVTWDX
+      CWSOWEVFWF $.
```

Comment:
No change in Syntax, Axioms and Definitions usage.  Proof length reduced from52 to 41 lines.